compiler/damlc/tests/daml-test-files/Interface.EXPECTED.md

```diff
@@ -64,26 +64,26 @@
 
 ## Functions
 
-<a name="function-interface-noopimpl-41891"></a>[noopImpl](#function-interface-noopimpl-41891)
-
-> : [Implements](https://docs.daml.com/daml/stdlib/Prelude.html#type-da-internal-interface-implements-92077) t [Token](#type-interface-token-10651) =\> t -\> () -\> [Update](https://docs.daml.com/daml/stdlib/Prelude.html#type-da-internal-lf-update-68072) ()
-
-<a name="function-interface-transferimpl-36342"></a>[transferImpl](#function-interface-transferimpl-36342)
+<a name="function-interface-getowner-36980"></a>[getOwner](#function-interface-getowner-36980)
```

Comment:
Since the declarations in an interface are now (un-)reversed in [`extractInterfaceBodyDecls`](https://github.com/digital-asset/ghc/pull/106/files#diff-e599e1c5f9a0d22ed9a8427c797d148bedca1d348e5abc9950490372287fc8f8R2259-R2260), this had to be reversed too. Now, the docs will show interface methods in the same order as they were written in Daml.

Reply:
sounds sensible :+1: